### PR TITLE
fix kube rbac proxy start error (#547)

### PR DESCRIPTION
**Pull request by ethan256**
@Allex1  The /home/nonroot directory gets created with 0700 permissions, which [means](http://www.filepermissions.com/file-permission/0700) that the owning user (nonroot) has Read, Write, and Execute permissions on the directory, but other users, even those in the same group, have no permissions. In order to chdir, the calling process must have Execute permissions on the target directory.The /home/nonroot directory gets created with 0700 permissions, which [means](http://www.filepermissions.com/file-permission/0700) that the owning user (nonroot) has Read, Write, and Execute permissions on the directory, but other users, even those in the same group, have no permissions. In order to chdir, the calling process must have Execute permissions on the target directory. For more information, please refer to [CHDIR TO CWD: PERMISSION DENIED](https://danielmangum.com/posts/runc-chdir-to-cwd/)

Therefore, it works fine with the following configuration
```shell
securityContext:
  runAsGroup: 65532
  runAsNonRoot: true
  runAsUser: 65532
  fsGroup: 65532
```

### CODE REVIEW

**Comment by Allex1** [comment]
@ethan256 you need to bump the chart , `make generate-examples` in order for the CI to pass

**Review by JaredTan95 — APPROVED** [review]
LGTM

**Comment by Allex1** [comment]
@ethan256 this looks good but you still need to bump the operator chart to 0.20.3 and omit/discard the opentelemetry-demo rendered files as I think they are leftovers from a different pr. cc @puckpuck

**Comment by JaredTan95** [comment]
yes, I didn't seen you re-rendered `opentelemetry-demo` example files.

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 TMP_DIRECTORY = ./tmp
-CHARTS ?= opentelemetry-collector opentelemetry-operator opentelemetry-demo
+CHARTS ?= opentelemetry-collector opentelemetry-operator
```
Comment (puckpuck) [inline Makefile:2]: Why is `opentelemetry-demo` removed from this list?

**Comment by JaredTan95** [comment]
@ethan256 hi, you can ref my PR's changes https://github.com/open-telemetry/opentelemetry-helm-charts/pull/563

File: charts/opentelemetry-collector/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.42.0
+version: 0.43.2
```
Comment (JaredTan95) [inline charts/opentelemetry-collector/Chart.yaml:3]: As other PR merged, you need to bump up again. :-P

**Review by JaredTan95 — CHANGES_REQUESTED** [review]

**Comment by JaredTan95** [comment]
and the last thing, pls resolve conflicts.

File: charts/opentelemetry-demo/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.15.1
+version: 0.15.2
```
Comment (Allex1) [inline charts/opentelemetry-demo/Chart.yaml:4]: this is unnecessary

File: charts/opentelemetry-operator/examples/default/rendered/tests/test-service-connection.yaml [hunk 4]
```diff
@@ -43,7 +43,11 @@ metadata:
   name: "opentelemetry-operator-webhook-test-connection"
   namespace: default
   labels:
+<<<<<<< HEAD
+    helm.sh/chart: opentelemetry-operator-0.20.4
+=======
     helm.sh/chart: opentelemetry-operator-0.20.3
```
Comment (Allex1) [inline charts/opentelemetry-operator/examples/default/rendered/tests/test-service-connection.yaml:49]: only `helm.sh/chart: opentelemetry-operator-0.20.4` should be here

**Review by Allex1 — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**